docs/proposals/node-agent/README.md

```diff
@@ -0,0 +1,41 @@
+# Node Agent
+
+## Requirements
+
+### Which service must the domain provide
+
+The node agent amanges workload management. A workload is the tiniest unit in orka. It can be a container, a virtual machine, etc.
+
+### What is in scope and out of scope
+
+#### In scope
+
+The Node Agent provides an API :
+
+- to control the workloads :
+  - start
+  - stop
+  - restart
+- health of node
+- metrics of nodes
+- status of workloads
+  - waiting
+  - running
+  - terminated
+- keeping workloads running/alive
+
+#### Out of scope
+
+- networking
```

Comment:
It's not entirely out of scope, as it's expecting some hooks into the networking provider/interface. What's out of scope is the networking configuration.